### DISCUSSION TITLE
How to implement fine-tuning for Chronos-Bolt

### DISCUSSION THREAD

**Discussion by shenxinyuan-hash** [amazon-science/chronos-forecasting#229] [Q&A] [answered]
How to implement fine-tuning for Chronos-Bolt, kindly request the author to update the yaml file and train.py

**Comment by abdulfatir** [comment, marked as answer]
@shenxinyuan-hash Thank you for your interest. We do not have immediate plans to add training/fine-tuning scripts for Chronos-Bolt to this repo. Please [use AutoGluon](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-chronos.html#fine-tuning), if you are interested in fine-tuning Chronos-Bolt.
